apps/test/unit/templates/CodeWorkspaceTest.js

```diff
@@ -17,7 +17,9 @@ describe('CodeWorkspace', () => {
     showProjectTemplateWorkspaceIcon: false,
     isMinecraft: false,
     runModeIndicators: false,
-    showMakerToggle: false
+    showMakerToggle: false,
+    displayWorkspaceAlert: false,
+    errorMsg: ''
```

Comment:
Might be good to add a few test cases here to verify the alert shows up as expected.

Reply:
Yes - I agree. Will do. Thanks!